restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -448,6 +455,77 @@ private DocumentDB maybeCreateTableAndIndexes(
     return idsWritten;
   }
 
+  public JsonNode executeBuiltInFunction(
```

Comment:
move to reactive doc service and make reactive, should be easier than mixing the two